Makefile

```diff
@@ -29,16 +29,18 @@ else
 	endif
 endif
 
-AUTOV8_VERSION = 7.4.288.28
+AUTOV8_VERSION = 8.8.278.14
 AUTOV8_DIR = build/v8
 AUTOV8_OUT = build/v8/out.gn/$(PLATFORM)/obj
 AUTOV8_DEPOT_TOOLS = build/depot_tools
 AUTOV8_LIB = $(AUTOV8_OUT)/libv8_snapshot.a
-AUTOV8_STATIC_LIBS = -lv8_base -lv8_snapshot -lv8_libplatform -lv8_libbase -lv8_libsampler
+AUTOV8_STATIC_LIBS = -lv8_base_without_compiler -lv8_compiler -lv8_base_without_compiler -lchrome_zlib -lcompression_utils_portable -ltorque_generated_definitions -lcppgc_base -lv8_cppgc_shared -lv8_snapshot -lv8_libplatform -lv8_libbase -lv8_libsampler -lchrome_zlib
```

Comment:
why not just use `v8_monolith` here?

Reply:
Can you list the options? I have failed to produce a build with no UND in symbol table... :)

Reply:
`use_custom_libcxx=false v8_monolithic=true v8_use_external_startup_data=false is_component_build=false`

and then build with:

`cd $(AUTOV8_DIR) ; env CXXFLAGS=-fPIC CFLAGS=-fPIC ninja -C out.gn/$(PLATFORM) v8_monolith`

this has worked well for me.

Reply:
Done!